.github/workflows/ci.yml

```
@@ -78,9 +78,9 @@ jobs:
         with:
           gh_token: ${{ secrets.GITHUB_TOKEN }}
           compile_mode: native
-          func: false
-          nistkat: false
-          kat: false
+          func: true
+          nistkat: true
+          kat: true
```

The expectation was that this would at least build the library, but it doesn't seem to be the case. The reason to this was to save work because the next job runs all tests with debug flag. 
```suggestion
          func: true
          nistkat: false
          kat: false
```

---

On second thought, the CI runtime seems ok even with your change, so let's keep that.